modules/runtime/nuxeo-runtime-jtajca/src/main/java/org/nuxeo/runtime/transaction/TransactionHelper.java

```diff
@@ -533,6 +534,23 @@ public static void registerSynchronization(Synchronization handler) {
         }
     }
 
+    /**
+     * Enlists a XA resource in the current transaction.
+     *
```

Comment:
is the method idempotent?

Reply:
No it's a direct link to the TransactionManager's identical method. In a previous version of the code it was used in many more places but now only one test (in `WorkManagerTXTest`) uses it... I think I'll keep it though as it doesn't do harm.